gulp/tasks/coverage.js

```diff
@@ -0,0 +1,14 @@
+(function() {
+    'use strict';
+
+    var gulp = require('gulp'),
+        coveralls = require('gulp-coveralls');
+
+    gulp.task('coverage', function() {
+        // Only run on Travis
+        if (process.argv.indexOf('--ci') !== -1) {
```

Comment:
@dsjen see Ben's comment below. It looks like I could probably just remove the if statement entirely. Does that seem fine?